count/Feature/ListeningQuizFeature.swift

```diff
@@ -433,22 +532,45 @@ struct ListeningQuizView: View {
 
     @ViewBuilder func progressBar(viewStore: ViewStoreOf<ListeningQuizFeature>) -> some View {
         HStack(spacing: 0) {
-            IndeterminateProgressView(
-                animationCount: viewStore.challengeCount,
-                color1: Color(.tintColor),
```

Comment:
This `tintColor` is way too prominent for the determinate variant, and I think it actually looks better as subtle gray anyway.